### ISSUE TITLE
Role broke after some of the last commits – AnsibleUndefinedVariable: One or more undefined variables: 'nginx_max_clients' is undefined

### ISSUE THREAD

**Issue by neverov** [jdauphant/ansible-role-nginx#53]
After the last commit I see failing provisions with error messages like this:

```
fatal: [52.11.27.223] => {'msg': "AnsibleUndefinedVariable: One or more undefined variables: 'nginx_max_clients' is undefined", 'failed': True}
```

I peeked [here](https://github.com/jdauphant/ansible-role-nginx/commit/5720a9d605e4a8ac90d0d246573a1d78ff19d0d7#diff-04c6e90faac2675aa89e2176d2eec7d8L22) and noticed that the `nginx_max_client` settings was removed, so I guess this might be the source of the problem?

**Comment by neverov** [comment]
Just to clarify, after going back to version 1.4.1 everything went back to normal.

**Comment by jdauphant** [comment]
Which version of Ansible do you use ?

**Comment by jdauphant** [comment]
Test are successfull with 1.9.1 : https://travis-ci.org/jdauphant/ansible-role-nginx/builds/63251494#L417

**Comment by neverov** [comment]
I use `ansible` in `circleci` builds, `ansible` is installed from `pip`, it looks like it's `1.9.1`, here's the log:
```
sudo pip install ansible boto
...
You are using pip version 6.0.8, however version 6.1.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
...
Collecting ansible
  Downloading ansible-1.9.1.tar.gz (917kB)
    100% |################################| 917kB 564kB/s 
...
```

Here's the role configuration I use:
```
...
- role: nginx
  sudo: yes
    nginx_http_params:
      - sendfile "on"
      - access_log "/var/log/nginx/access.log"
      - proxy_cache_path "/var/cache/nginx" levels=1:2 keys_zone=all:10m
    nginx_sites:
      backend:
        - listen 80
        - location / { proxy_pass http://localhost:5000; }         
         ... more nginx locations omitted
    nginx_configs:
      proxy:
        - proxy_set_header X-Real-IP  $remote_addr
        - proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for
...
```

**Comment by jdauphant** [comment]
I have made a PR to correct the bug, https://github.com/jdauphant/ansible-role-nginx/pull/54
I am waiting for the test.

**Comment by neverov** [comment]
Great, thanks for the fix!

**Comment by jdauphant** [comment]
Travis have some problem.
If you have test it and you can confirm this fix is ok, I will merge and create a version on Galaxy.

Thanks in advance,
Julien

**Comment by jdauphant** [comment]
Fix released in 1.5.1
